card/VERSION

```diff
@@ -1 +1 @@
-1.14.8s
+1.15.pre
```

Comment:
Actually, I think it needs to stay here (I made that comment before I'd finished tidying up).  The issue is that the card gem actually needs direct access to the version from the self/version card, and that has to work in a normal deployed gem environment.  In that environment, the base directory doesn't even exist ;)

There could be other solutions for that, but let's make sure we don't break that case!